subprojects/core-api/src/main/java/org/gradle/api/attributes/java/package-info.java

```diff
@@ -17,5 +17,4 @@
 /**
  * Attributes specific to the Java ecosystem.
  */
-@org.gradle.api.NonNullApi
```

Comment:
Why was this removed, especially from a public API package? This might be a breaking change for Kotlin